tools/sof-kernel-log-check.sh

```diff
@@ -365,6 +365,11 @@ ignore_str="$ignore_str"'|elan_i2c i2c-ELAN0000:.*: invalid report id data'
 # iwlwifi 0000:00:14.3: No beacon heard and the time event is over already...
 ignore_str="$ignore_str"'|iwlwifi [[:digit:].:]+: '
 
+# Ignore DEVRES debug log which is ERROR by default
+# DEVRES debug log will be enabled in kmod test
+# There are four OPs from DEVRES, ADD, REP, REM, REL
+# for example, [  182.317913] cml_rt1011_rt5682 cml_rt1011_rt5682: DEVRES REL 00000000618e4fb0 devm_card_release (8 bytes)
+ignore_str="$ignore_str"'|DEVRES '
```

Comment:
I think we can start by being a bit more specific:
```suggestion
ignore_str="$ignore_str"'| DEVRES [A-Z]{3,}'
```

Reply:
OP prints 3 characters only. 
```
dev_err(dev, "DEVRES %3s %p %s (%zu bytes)\n",
			op, node, node->name, node->size);
```
I updated like this,
`ignore_str="$ignore_str"'| DEVRES [A-Z]{3} '`

Reply:
There is also REPLACE, 7 characters long. This won't catch it.

Reply:
no, for replace, it print REP. OP prints only 3 characters.

Reply:
[dmesg.txt.gz](https://github.com/thesofproject/sof-test/files/6735086/dmesg.txt.gz)
I attached dmesg output from a kmod test with DEVRES.